index.html

```diff
@@ -93,16 +93,17 @@ <h2>Communication and organisation within the community:</h2>
 <p>An open communication culture prevails. Although we develop and use Samply in a variety of independent research projects, we are committed to communicating ideas, problems and changes openly.</p>
 <p>For discussions, questions, complaints or feature requests, contact the developers via the <a href="mailto:dev@samply.de">Samply mailing list</a>, <a href="https://github.com/samply">Issue Tracker</a> or <a href="https://chat.germanbiobanknode.de">Samply chat</a>.</p>
 <p>Our working language for documentation is English. In meetings, we will choose our working language based on the preferences of the attendees.</p>
-<p>Members of the Samply community with long-term activities in developing Samply software may become members of the Samply committee, either by invitation from the current committee or by requesting admission.</p>
-<p>The members of the Samply committee decide by a majority about organisational topics like changes to the internal workflow. Modifications to the Samply manifest must be agreed unanimously by the committee members.</p>
+<p>The Samply community is represented by the Samply committee. The members of the Samply committee decide by a majority about organisational topics like changes to the internal workflow. Modifications to the Samply manifest must be agreed unanimously by the committee members.</p>
+<p>The Samply committee consists of at least five, and at most nine members. If possible, at least two different active projects associated with the Samply software should be represented in the Samply committee.</p>
+<p>Current committee members may propose new committee members. Alternatively, community members may request admission to the committee. In both cases, the committee decides about the admission to the committee. Each committee member is elected for 12 months. Re-election is possible.</p>

```

Comment:
How does the election work?

Reply:
I'd prefer to keep it simple in the manifest. In your opinion, would it suffice to say it has to be decided with a x% majority? If yes, what would be an appropriate percentage? I'd suggest 75%.

Reply:
In my opinion, the election of the committee is central. Can you copy something from Kubernetes? Also if a new member joins the committee, who goes if the committee as already 9 members?

Reply:
Reference Kubernetes Steering Committee Elections: https://github.com/kubernetes/steering/blob/master/elections.md

Eligibility for voting:
- eople who had at least 50 contributions to the Kubernetes project over the past year....
- People who have submitted the voting exception form and are accepted by the election committee...

Eligibility for candidacy:
- Acceptance of a nomination, or self-nomination (anyone may nominate, anyone may be nominated)
- Endorsement by three eligible voters from three different employers (the candidate can self-endorse if they are eligible to vote)

- Guide for details: https://github.com/kubernetes/community/tree/master/events/elections/2019#candidacy-process

Election process
- Elections will be held using time-limited Condorcet ranking on CIVS using the IRV method. The top vote getters will be elected to the open seats.
- To encourage diversity there will be a maximum of one-third representation on the Steering Committee from any one company at any time. If the results of an election result in greater than 1/3 representation, the lowest vote getters from any particular company will be removed until representation on the committee is less than one-third.
- If percentages shift because of job changes, acquisitions, or other events, sufficient members of the committee must resign until max one-third representation is achieved. If it is impossible to find sufficient members to resign, the entire company’s representation will be removed and new special elections held. In the event of a question of company membership (for example evaluating independence of corporate subsidiaries) a majority of all non-involved Steering Committee members will decide.

Reply:
What exactly does three employers mean?

But moreover, I do not think that a Pullrequest (Code Review) is the best place to discuss this topic.